juseppe-cli/src/main/java/ru/lanwen/jenkins/juseppe/files/WatchFiles.java

```diff
@@ -18,60 +26,121 @@
 import static ru.lanwen.jenkins.juseppe.files.WatchEventExtension.hasExt;
 
 /**
- * User: lanwen
- * Date: 26.01.15
- * Time: 12:43
+ * User: lanwen Date: 26.01.15 Time: 12:43
  */
 public class WatchFiles extends Thread {
 
-    private static final Logger LOG = LoggerFactory.getLogger(WatchFiles.class);
-
-    private WatchService watcher;
-    private Path path;
-    private Props props;
-
-    private WatchFiles() {
-        setDaemon(true);
-    }
-
-    public WatchFiles configureFor(Props props) throws IOException {
-        this.props = props;
-        path = Paths.get(props.getPluginsDir());
-        setName(format("file-watcher-%s", path.getFileName()));
-
-
-        watcher = this.path.getFileSystem().newWatchService();
-        path.register(watcher,
-                ENTRY_CREATE,
-                ENTRY_DELETE,
-                ENTRY_MODIFY
-        );
-        return this;
-    }
-
-    public static WatchFiles watchFor(Props props) throws IOException {
-        return new WatchFiles().configureFor(props);
-    }
-
-    @Override
-    public void run() {
-        LOG.info("Start to watch for changes: {}", path);
-        try {
-            // get the first event before looping
-            WatchKey key = watcher.take();
-            while (key != null) {
-
-                if (key.pollEvents().stream().anyMatch(hasExt(".hpi").or(hasExt(".jpi")))) {
-                    LOG.trace("HPI (JPI) list modify found!");
-                    UpdateSiteGen.updateSite(props).withDefaults().toSave().saveAll();
-                }
-
-                key.reset();
-                key = watcher.take();
+   private static final Logger LOG = LoggerFactory.getLogger(WatchFiles.class);
```

Comment:
please avoid indentation changes - it is hard to review actual changes in this case...